.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Run unit tests
+
+on: [pull_request]
+
+jobs:
+  unit-tests:
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        id: setup-python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+          architecture: x64
```

Comment:
fyi may be worth it to set up a matrix test with multiple python versions: https://github.com/feast-dev/feast/blob/master/.github/workflows/unit_tests.yml

Reply:
Yes agree https://github.com/Shopify/feast-trino/issues/21